pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -707,6 +667,16 @@ func (b *Build) setupSecretVolumesAndArgs(secrets []corev1.Secret, filter func(s
 		case secret.Type == corev1.SecretTypeSSHAuth:
 			annotatedUrl := secret.Annotations[GITSecretAnnotationPrefix]
 			args = append(args, fmt.Sprintf("-ssh-%s=%s=%s", "git", secret.Name, annotatedUrl))
+		case string(secret.Data[COSIGNSecretDataCosignKey]) != "":
+			if cosignRepository := secret.ObjectMeta.Annotations[COSIGNRespositoryAnnotationPrefix]; cosignRepository != "" {
+				args = append(args, fmt.Sprintf("-cosign-repositories=%s=%s", secret.Name, cosignRepository))
+			}
+
+			if cosignDockerMediaType := secret.ObjectMeta.Annotations[COSIGNDockerMediaTypesAnnotationPrefix]; cosignDockerMediaType != "" {
+				args = append(args, fmt.Sprintf("-cosign-docker-media-types=%s=%s", secret.Name, cosignDockerMediaType))
+			}
+
```

Comment:
For readability purposes, could this be extracted to a separate function/method?